web/src/components/pages/profile/avatar-setup/avatar-setup.tsx

```diff
@@ -15,6 +15,7 @@ import { Uploads } from '../../../../stores/uploads';
 import { CheckIcon, LinkIcon } from '../../../ui/icons';
 
 import './avatar-setup.css';
+import { Spinner } from '../../../ui/ui';
```

Comment:
Style - not important, could we move these dependancies above the CSS import with the others?